src/cases/containers/query-management-container/query-management-container.component.ts

```diff
@@ -123,13 +176,31 @@ export class QueryManagementContainerComponent implements OnInit {
     window.scrollTo({ left: 0, top: 0, behavior: 'smooth' });
   }
 
-  public navigateToErrorElement(elementId: string): void {
-    if (elementId) {
-      const htmlElement = document.getElementById(elementId);
-      if (htmlElement) {
-        htmlElement.scrollIntoView({ behavior: 'smooth', block: 'center' });
-        htmlElement.focus();
-      }
+  private getQueryCreateContext(queryItemId: string): QueryItemType {
+    if (!queryItemId) {
```

Comment:
Is this conditon required? Will this not be handled within switch default?

Reply:
@DavidJayakumar it is part of the code I wrote for https://tools.hmcts.net/jira/browse/EUI-8523 and yes you are correct, the default is hit when the `queryItemId` is undefined.

I will handle this as part of my current ticket https://tools.hmcts.net/jira/browse/EUI-8524.